src/main/java/server/ProxyServer.java

```diff
@@ -65,15 +116,18 @@ public TokenIO initializeSdk() throws IOException {
      * @return newly-created member
      */
     private Member createMember(TokenIO tokenIO) {
```

Comment:
I'd move all of these private members into a separate class. TokenClient? You have one private method per one public method. The ProxyServer then becomes about extracting request parameters and generating response protos. And the TokenClient is the client that you use to carry out the requests.

Reply:
OK